sdk/containerregistry/service.projects

```diff
@@ -31,8 +36,4 @@
     <ProjectReference Remove="@(ProjectsToRemove)" />
   </ItemGroup>
```

Comment:
I'd add the condition on this ItemGroup to essentially not do this Remove if SDKType=all

Reply:
Moved even more logic into the service.proj